### ISSUE TITLE
[DragonAPI] Instantiable Event Ticking Screen Crash

### ISSUE THREAD

**Issue by RockmanSolid** [ReikaKalseki/Reika_Mods_Issues#3071]
Added the API to my server modpack to solve potion ID issues and everything worked great! Then, when my girlfriend's fire resistance potion ran out, she crashed and hasn't been able to come back on since. The issue that comes up is linked to the API, but I can't seem to find evidence of it or anything like it anywhere. I am unaffected. The following is her crash report. Any guidance would be heavily appreciated!

---- Minecraft Crash Report ----
// Who set us up the TNT?

Time: 9/2/22 4:17 AM
Description: Ticking screen

java.lang.NullPointerException: Ticking screen
	at Reika.DragonAPI.Instantiable.Event.ApplyPotionEvent.fire(ApplyPotionEvent.java:47)
	at net.minecraft.entity.EntityLivingBase.func_70690_d(EntityLivingBase.java:632)
	at net.minecraft.client.network.NetHandlerPlayClient.func_147260_a(NetHandlerPlayClient.java:1340)
	at net.minecraft.network.play.server.S1DPacketEntityEffect.func_148833_a(SourceFile:55)
	at net.minecraft.network.play.server.S1DPacketEntityEffect.func_148833_a(SourceFile:12)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.client.multiplayer.GuiConnecting.func_73876_c(SourceFile:78)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1661)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at Reika.DragonAPI.Instantiable.Event.ApplyPotionEvent.fire(ApplyPotionEvent.java:47)
	at net.minecraft.entity.EntityLivingBase.func_70690_d(EntityLivingBase.java:632)
	at net.minecraft.client.network.NetHandlerPlayClient.func_147260_a(NetHandlerPlayClient.java:1340)
	at net.minecraft.network.play.server.S1DPacketEntityEffect.func_148833_a(SourceFile:55)
	at net.minecraft.network.play.server.S1DPacketEntityEffect.func_148833_a(SourceFile:12)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.client.multiplayer.GuiConnecting.func_73876_c(SourceFile:78)

-- Affected screen --
Details:
	Screen name: ~~ERROR~~ NullPointerException: null

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityClientPlayerMP['Nytaris'/221975, l='MpServer', x=-1239.37, y=74.62, z=-690.77]]
	Chunk stats: MultiplayerChunkCache: 0, 0
	Level seed: 0
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: World: (92,64,248), Chunk: (at 12,4,8 in 5,15; contains blocks 80,0,240 to 95,255,255), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 12324318 game time, 14655795 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Forced entities: 1 total; [EntityClientPlayerMP['Nytaris'/221975, l='MpServer', x=-1239.37, y=74.62, z=-690.77]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:373)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2444)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:919)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 795768288 bytes (758 MB) / 1174405120 bytes (1120 MB) up to 6442450944 bytes (6144 MB)
	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx6G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1614 73 mods loaded, 73 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCHIJA	Forge{10.13.4.1614} [Minecraft Forge] (forge-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCHIJA	AM2-Preloader{0.0.3} [AMCore] (minecraft.jar) 
	UCHIJA	appliedenergistics2-core{rv3-beta-5} [Applied Energistics 2 Core] (minecraft.jar) 
	UCHIJA	CodeChickenCore{1.0.7.48} [CodeChicken Core] (minecraft.jar) 
	UCHIJA	MobiusCore{1.2.5} [MobiusCore] (minecraft.jar) 
	UCHIJA	NotEnoughItems{1.0.5.120} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.5.120-universal.jar) 
	UCHIJA	ThE-core{1.0.0.1} [Thaumic Energistics Core] (minecraft.jar) 
	UCHIJA	ThaumicTinkerer-preloader{0.1} [Thaumic Tinkerer Core] (minecraft.jar) 
	UCHIJA	xaerominimap_core{1.7.10-1.0} [XaeroMinimapCore] (minecraft.jar) 
	UCHIJA	xaeroworldmap_core{1.7.10-1.0} [XaeroWorldMapCore] (minecraft.jar) 
	UCHIJA	OpenModsCore{0.10.1} [OpenModsCore] (minecraft.jar) 
	UCHIJA	<CoFH ASM>{000} [CoFH ASM] (minecraft.jar) 
	UCHIJA	<DragonAPI ASM>{0} [DragonAPI ASM Data Initialization] (minecraft.jar) 
	UCHIJA	FoamFixCore{1.0.0} [FoamFixCore] (minecraft.jar) 
	UCHIJA	AnimationAPI{1.2.4} [AnimationAPI] (AnimationAPI-1.7.10-1.2.4.jar) 
	UCHIJA	arsmagica2{1.4.0.009} [Ars Magica 2] (1.7.10_AM2-1.4.0.009.jar) 
	UCHIJA	adventurebackpack{1.7.10-0.9-h12} [Adventure Backpack] (adventurebackpack-1.7.10-0.9-h12.jar) 
	UCHIJA	appliedenergistics2{rv3-beta-5} [Applied Energistics 2] (appliedenergistics2-rv3-beta-5.jar) 
	UCHIJA	bdlib{1.9.4.109} [BD Lib] (bdlib-1.9.4.109-mc1.7.10.jar) 
	UCHIJA	ae2stuff{0.5.1.9} [AE2 Stuff] (ae2stuff-rv3-0.5.1.9-mc1.7.10.jar) 
	UCHIJA	AncientWarfare{2.4.114-beta-MC1.7.10} [Ancient Warfare Core] (ancientwarfare-2.4.115-beta-MC1.7.10-FULL.jar) 
	UCHIJA	CoFHCore{1.7.10R3.1.4} [CoFH Core] (CoFHCore-[1.7.10]3.1.4-329.jar) 
	UCHIJA	AncientWarfareAutomation{2.4.114-beta-MC1.7.10} [Ancient Warfare Automation] (ancientwarfare-2.4.115-beta-MC1.7.10-FULL.jar) 
	UCHIJA	AncientWarfareNEIPlugin{2.4.114-beta-MC1.7.10} [Ancient Warfare NEI Plugin] (ancientwarfare-2.4.115-beta-MC1.7.10-FULL.jar) 
	UCHIJA	AncientWarfareNpc{2.4.114-beta-MC1.7.10} [Ancient Warfare NPCs] (ancientwarfare-2.4.115-beta-MC1.7.10-FULL.jar) 
	UCHIJA	AncientWarfareStructure{2.4.114-beta-MC1.7.10} [Ancient Warfare Structures] (ancientwarfare-2.4.115-beta-MC1.7.10-FULL.jar) 
	UCHIJA	AppleCore{3.1.1} [AppleCore] (AppleCore-mc1.7.10-3.1.1.jar) 
	UCHIJA	BiblioCraft{1.11.7} [BiblioCraft] (BiblioCraft[v1.11.7][MC1.7.10].jar) 
	UCHIJA	AWWayofTime{v1.3.3} [Blood Magic: Alchemical Wizardry] (BloodMagic-1.7.10-1.3.3-17.jar) 
	UCHIJA	Baubles{1.0.1.10} [Baubles] (Baubles-1.7.10-1.0.1.10.jar) 
	UCHIJA	Thaumcraft{4.2.3.5} [Thaumcraft] (Thaumcraft-1.7.10-4.2.3.5.jar) 
	UCHIJA	Botania{r1.8-249} [Botania] (Botania r1.8-249.jar) 
	UCHIJA	TwilightForest{2.3.8dev} [The Twilight Forest] (twilightforest-1.7.10-2.3.8.jar) 
	UCHIJA	chisel{2.9.5.11} [Chisel] (Chisel-2.9.5.11.jar) 
	UCHIJA	CosmeticArmor{1.0.5} [Cosmetic Armor] (CosmeticArmor-1.7.10-1.0.5.B8-universal.jar) 
	UCHIJA	numina{1.7.10} [Numina] (Numina-0.4.1.106.jar) 
	UCHIJA	p455w0rdslib{1.0.4} [p455w0rd's Library] (p455w0rdslib-1.7.10-1.0.4.jar) 
	UCHIJA	ae2wct{1.7.10-rv3-1.8.7.4b} [AE2 Wireless Crafting Terminal] (WirelessCraftingTerminal-1.7.10-rv3-1.8.7.4b.jar) 
	UCHIJA	powersuits{1.7.10-0.11.1.117} [MachineMuse's Modular Powersuits] (ModularPowersuits-1.7.10-0.11.1.117.jar) 
	UCHIJA	ThermalFoundation{1.7.10R1.2.6} [Thermal Foundation] (ThermalFoundation-[1.7.10]1.2.6-118.jar) 
	UCHIJA	ThermalExpansion{1.7.10R4.1.5} [Thermal Expansion] (ThermalExpansion-[1.7.10]4.1.5-248.jar) 
	UCHIJA	Waila{1.5.10} [Waila] (Waila-1.5.10_1.7.10.jar) 
	UCHIJA	DragonAPI{v31b} [DragonAPI] (DragonAPI1.7.10V31b.jar) 
	UCHIJA	dsurround{1.0.6.4} [Dynamic Surroundings] (DynamicSurroundings-1.7.10-1.0.6.4.jar) 
	UCHIJA	EnderStorage{1.4.7.37} [EnderStorage] (EnderStorage-1.7.10-1.4.7.37-universal.jar) 
	UCHIJA	ExtraUtilities{1.2.12} [Extra Utilities] (extrautilities-1.2.12.jar) 
	UCHIJA	foamfix{@VERSION@} [FoamFix] (FoamFix-1.7.10-universal-1.0.0.jar) 
	UCHIJA	Steamcraft{0.29.3} [Flaxbeard's Steam Power] (FSP-1.7.10-0.29.3.jar) 
	UCHIJA	iChunUtil{4.2.3} [iChunUtil] (iChunUtil-4.2.3.jar) 
	UCHIJA	inventorytweaks{1.59-dev-152-cf6e263} [Inventory Tweaks] (InventoryTweaks-1.59-dev-152.jar) 
	UCHIJA	IronChest{6.0.62.742} [Iron Chest] (ironchest-1.7.10-6.0.62.742-universal.jar) 
	UCHIJA	malisiscore{1.7.10-0.14.3} [MalisisCore] (malisiscore-1.7.10-0.14.3.jar) 
	UCHIJA	malisisdoors{1.7.10-1.13.2} [Malisis' Doors] (malisisdoors-1.7.10-1.13.2.jar) 
	UCHIJA	MouseTweaks{2.4.4} [Mouse Tweaks] (MouseTweaks-2.4.4-mc1.7.10.jar) 
	UCHIJA	OpenMods{0.10.1} [OpenMods] (OpenModsLib-1.7.10-0.10.1.jar) 
	UCHIJA	OpenBlocks{1.6} [OpenBlocks] (OpenBlocks-1.7.10-1.6.jar) 
	UCHIJA	oreexcavation{1.1.134} [OreExcavation] (OreExcavation-1.1.134.jar) 
	UCHIJA	StorageDrawers{1.7.10-1.10.9} [Storage Drawers] (StorageDrawers-1.7.10-1.10.9.jar) 
	UCHIJA	Sync{4.0.1} [Sync] (Sync-4.0.1.jar) 
	UCHIJA	tcinventoryscan{1.0.11} [TC Inventory Scanning] (tcinventoryscan-mc1.7.10-1.0.11.jar) 
	UCHIJA	tcnodetracker{1.1.2} [TC Node Tracker] (tcnodetracker-1.7.10-1.1.2.jar) 
	UCHIJA	thaumcraftneiplugin{@VERSION@} [Thaumcraft NEI Plugin] (thaumcraftneiplugin-1.7.10-1.7a.jar) 
	UCHIJA	thaumicenergistics{1.1.3.0} [Thaumic Energistics] (thaumicenergistics-1.1.3.0.jar) 
	UCHIJA	ThaumicTinkerer{unspecified} [Thaumic Tinkerer] (ThaumicTinkerer-2.5-1.7.10-164.jar) 
	UCHIJA	ThaumicExploration{0.6.0} [Thaumic Exploration] (ThaumicExploration-1.7.10-1.1-53.jar) 
	UCHIJA	ThermalDynamics{1.7.10R1.2.1} [Thermal Dynamics] (ThermalDynamics-[1.7.10]1.2.1-172.jar) 
	UCHIJA	wawla{1.3.1} [What Are We Looking At] (Wawla-1.0.5.120.jar) 
	UCHIJA	witchery{0.24.1} [Witchery] (witchery-1.7.10-0.24.1.jar) 
	UCHIJA	XaeroMinimap{21.10.27} [Xaero's Minimap] (Xaeros_Minimap_21.10.27_Forge_1.7.10.jar) 
	UCHIJA	XaeroWorldMap{1.14.1.20} [Xaero's World Map] (XaerosWorldMap_1.14.1.20_Forge_1.7.10.jar) 
	UCHIJA	UniDict{1.7.10-2.9.2} [UniDict] (UniDict-1.7.10-2.9.2.jar) 
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.6.0 NVIDIA 472.19' Renderer: 'NVIDIA GeForce GTX 1660 Ti/PCIe/SSE2'
	OpenModsLib class transformers: [stencil_patches:FINISHED],[movement_callback:FINISHED],[player_damage_hook:FINISHED],[map_gen_fix:FINISHED],[gl_capabilities_hook:FINISHED],[player_render_hook:FINISHED]
	Class transformer null safety: found misbehaving transformers: me.guichaguri.betterfps.transformers.MathTransformer(me.guichaguri.betterfps.transformers.MathTransformer@69d9b9f5) returned non-null result: 0,me.guichaguri.betterfps.transformers.EventTransformer(me.guichaguri.betterfps.transformers.EventTransformer@5f6ffaa8) returned non-null result: 0
	AE2 Version: beta rv3-beta-5 for Forge 10.13.4.1448
	CoFHCore: -[1.7.10]3.1.4-329
	ThermalFoundation: -[1.7.10]1.2.6-118
	ThermalExpansion: -[1.7.10]4.1.5-248
	ThermalDynamics: -[1.7.10]1.2.1-172
	List of loaded APIs: 
		* ae2wct|API (1.7.10-rv3-1.8.6b) from WirelessCraftingTerminal-1.7.10-rv3-1.8.7.4b.jar
		* AgriCraftAPI (1.0) from DragonAPI1.7.10V31b.jar
		* AncientWarfareAPI (1.0) from ancientwarfare-2.4.115-beta-MC1.7.10-FULL.jar
		* AppleCoreAPI (3.1.0) from AppleCore-mc1.7.10-3.1.1.jar
		* appliedenergistics2|API (rv3) from appliedenergistics2-rv3-beta-5.jar
		* Baubles|API (1.0.1.10) from Baubles-1.7.10-1.0.1.10.jar
		* BloodMagicAPI (1.3.3-13) from BloodMagic-1.7.10-1.3.3-17.jar
		* BotaniaAPI (76) from Botania r1.8-249.jar
		* BuildCraftAPI|core (1.0) from extrautilities-1.2.12.jar
		* BuildCraftAPI|tools (1.0) from extrautilities-1.2.12.jar
		* ChiselAPI (0.1.1) from Chisel-2.9.5.11.jar
		* ChiselAPI|Carving (0.1.1) from Chisel-2.9.5.11.jar
		* ChiselAPI|Rendering (0.1.1) from Chisel-2.9.5.11.jar
		* CoFHAPI (1.7.10R1.0.13B1) from extrautilities-1.2.12.jar
		* CoFHAPI|block (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|core (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|energy (1.7.10R1.0.13B2) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHAPI|fluid (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|inventory (1.7.10R1.0.13B2) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHAPI|item (1.7.10R1.0.13B2) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHAPI|modhelpers (1.7.10R1.0.13B2) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHAPI|tileentity (1.7.10R1.0.13B2) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHAPI|transport (1.7.10R1.0.13B2) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHAPI|world (1.7.10R1.0.13B2) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|audio (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|gui (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|gui|container (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui|element (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|gui|element|listbox (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|gui|slot (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|inventory (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|render (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|render|particle (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|util (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|util|helpers (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|util|position (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|world (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* CoFHLib|world|feature (1.7.10R1.0.3B3) from CoFHLib-[1.7.10]1.2.1-185.jar
		* minechemAPI (1.7.10R1) from DragonAPI1.7.10V31b.jar
		* OpenBlocks|API (1.1) from OpenBlocks-1.7.10-1.6.jar
		* StorageDrawersAPI (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|config (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|event (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|inventory (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|pack (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|registry (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|render (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|storage (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|storage-attribute (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* Thaumcraft|API (4.2.2.0) from Thaumcraft-1.7.10-4.2.3.5.jar
		* thaumicenergistics|API (1.1) from thaumicenergistics-1.1.3.0.jar
		* WailaAPI (1.2) from Waila-1.5.10_1.7.10.jar
	Chisel: Errors like "[FML]: Unable to lookup ..." are NOT the cause of this crash. You can safely ignore these errors. And update forge while you're at it.
	Stencil buffer state: Function set: GL30, pool: forge, bits: 8
	AE2 Integration: IC2:OFF, RotaryCraft:OFF, RC:OFF, BuildCraftCore:OFF, BuildCraftTransport:OFF, BuildCraftBuilder:OFF, RF:ON, RFItem:ON, MFR:OFF, DSU:ON, FZ:OFF, FMP:OFF, RB:OFF, CLApi:OFF, Waila:ON, InvTweaks:ON, NEI:ON, CraftGuide:OFF, Mekanism:OFF, ImmibisMicroblocks:OFF, BetterStorage:OFF, OpenComputers:OFF, PneumaticCraft:OFF
	Launched Version: 1.7.10-Forge10.13.4.1614-1.7.10
	LWJGL: 2.9.1
	OpenGL: NVIDIA GeForce GTX 1660 Ti/PCIe/SSE2 GL version 4.6.0 NVIDIA 472.19, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: []
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Anisotropic Filtering: Off (1)

**Comment by RockmanSolid** [comment]
After running /effect [PlayerName] clear, that has cleared the issue, but I'll leave this up in case another solution can be found.

**Comment by Twisted-Code** [comment]
I encountered this issue when I tried to put on Leggings of the Burning Mantle from Thaumic Tinker while DragonAPI (and also RotaryCraft but I'm not sure if that's relevant) were installed. Normally, this endgame TT armor gives you a permanent fire resistance potion and causes fire to heal you instead of damaging... but with DragonAPI installed it seems to just cause a client crash instead. Probably will continue to do so until both the potion effect and the armor applying it are removed...
Surely we can come up with a better solution to this than "just don't get fire resistance"?

**Comment by Twisted-Code** [comment]
(Just to be clear that this is the same issue, here's my crash report. Very similar stacktrace)
[crash-2022-09-18_21.39.26-client.txt](https://github.com/ReikaKalseki/Reika_Mods_Issues/files/9595752/crash-2022-09-18_21.39.26-client.txt)

**Comment by Twisted-Code** [comment]
Strangely, it just happened again on a completely new world when I wasn't wearing any armor. Instead it seems like it happened in response to some GUI event. I was fine when I first spawned in, then when I opened my inventory I was fine... until I tried to search for "mantle" in order to display the leggings I mentioned above.
Now any time I open my inventory in that world, I instantly crash. It's something to do with displaying the inventory or possibly NEI. Could it have something to do with the in-inventory status effect display? Except I didn't have any status effect when I crashed this second time...
I'm at a loss for what precisely is causing this. You know your code better than I do Reika. What is this [ApplyPotionEvent.fire](https://github.com/ReikaKalseki/DragonAPI/blob/master/Instantiable/Event/ApplyPotionEvent.java) method "supposed to do" exactly? it's a handler for "player is on lit on fire" right? Why is something trying to light me on fire (judging by the name, though I suppose it can also be fire resistance based on the first crash) just from me looking at my inventory? And why is DragonAPI "[N[o]PEing out](https://docs.oracle.com/javase/10/docs/api/java/lang/NullPointerException.html)" when it tries to handle that event?

**Comment by Twisted-Code** [comment]
@RockmanSolid were you by any chance using Potion ID Helper when you were trying to fix potion IDs as you said?
I had the same issue when I had both that and DragonAPI enabled, but I think it's mostly on Potion ID Helper's side of the fence. According to its overview, it uses some ASM hacks to do what it does, and I think that's what is causing us to have this problem. Or at least, that's what caused it in my case. I had forgotten to disable Potion ID Helper like I intended to (I knew it was probably going to cause bugs/crashes if left enabled), and realized what was wrong when I experienced an NPE, from the same armor set, even after disabling DragonAPI (and of course RC which depends on DragonAPI).
So far at least, I have not had any further issues wearing the armor that was triggering my crash.
